docs/release-notes/known-issues.md

```diff
@@ -17,7 +17,29 @@ Known issues are significant defects or limitations that may impact your impleme
 
 % **Workaround**
 % Workaround description.
+% :::
 
+:::{dropdown} APM Integration / Server might get removed after upgrading to 8.19.0 and 9.1.0
```

Comment:
"APM Integration / Server might get removed" is not very clear to me. Pending confirmation on symptom description. Related: https://github.com/elastic/fleet-server/issues/5170